README.md

```diff
@@ -136,6 +136,8 @@ python -m venv .venv
 source .venv/bin/activate
 ```
 
+Note: If you have a Python3 version installed, you should try the command ```python3 -m venv .venv``` and then ```souce .venv/bin/activate```
```

Comment:
You could change this line so it could be copied like a code, breaking the line before and after the commands:
Note: If you have a Python3 version installed, you should try the command 
```python3 -m venv .venv``` 
and then 
```souce .venv/bin/activate```

Reply:
I thought not to show too much repetitive information

Maybe we can do this way:
Note: If you have a Python3 version installed, you should try the command
```
python3 -m venv .venv
```
...and then ```souce .venv/bin/activate``` command.

Reply:
this is good as well!